src/test/java/seedu/address/commons/util/CollectionUtilTest.java

```
@@ -107,6 +98,46 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    private void assertNullPointerExceptionThrown(Object... objects) {
+        try {
+            requireNonNull(objects);
+            fail();
+        } catch (NullPointerException npe) {
+        } catch (Exception e) {
+            fail();
```

No need for this, just let the exception throw.